source/Calamari.Common/Plumbing/FileSystem/CalamariPhysicalFileSystem.cs

```diff
@@ -201,22 +201,18 @@ IEnumerable<FileSystemInfo> EnumerateWithGlob(string parentDirectoryPath, params
                 .Select(g => g.First());
         }
 
-        public virtual IEnumerable<string> EnumerateFiles(string parentDirectoryPath, params string[] searchPatterns)
+        public virtual IEnumerable<string> EnumerateFiles(
+            string parentDirectoryPath,
+            params string[] searchPatterns)
         {
-            var parentDirectoryInfo = new DirectoryInfo(parentDirectoryPath);
-
-            return searchPatterns.Length == 0
-                ? parentDirectoryInfo.GetFiles("*", SearchOption.TopDirectoryOnly).Select(fi => fi.FullName)
-                : searchPatterns.SelectMany(pattern => parentDirectoryInfo.GetFiles(pattern, SearchOption.TopDirectoryOnly).Select(fi => fi.FullName));
+            return Directory.EnumerateFiles(parentDirectoryPath, searchPatterns);
```

Comment:
Does `Directory.EnumerateFiles(..)` actually accept an array of search patterns here?  I can't find it in the docs.

Reply:
Looking below, is the intent here to use `StandardDirectory.EnumerateFiles?`

Reply:
It's super confusing in a code review because `Dictionary` here is referring to a property on `CalamariPhysicalFileSystem` which is of type `IDictionary` and is an instance of `StandardDirectory` or `LongPathsDirectory` depending on the runtime of the Calamari.

So it's not just the `Dictionary` static method...

Reply:
Ah gotcha 👍

Reply:
*Directory

They both start with D so they are interchangeable right? 🤦‍♂️